### DISCUSSION TITLE
Broken peacock

### DISCUSSION THREAD

**Discussion by felixcrazzolara** [idaholab/moose#26550] [Q&A General] [answered]
Hello everyone,
Did anyone else ever suffer from a broken peakcock visualizer? Since day one of me using Moose, when switching between the tabs of peacock, the application occasionally break. You can see an example below:
![Selection_031](https://github.com/idaholab/moose/assets/28649168/aee2d785-60a9-45f1-af14-532828af9e63)
After starting the application it looks like this (as it should):
![Selection_032](https://github.com/idaholab/moose/assets/28649168/39c2f025-d398-45fa-bb64-3e0b61c60462)

In the terminal, I get the following errors:
```
QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 111, 111, 111, 255'
QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 180, 180, 180, 255'
QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 111, 111, 111, 255'
QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 111, 111, 111, 255'
QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 180, 180, 180, 255'
QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 111, 111, 111, 255'
```

**Comment by GiudGiud** [comment, marked as answer]
Hello

I have seen these rgb messages before I think they are just warnings.

What platform are you using? Can you paste the output of the diagnostics script in moose/scripts ?

Guillaume

**Reply by milljm** [reply]
This Discussion should have been marked as answered by @GiudGiud, as the issue is not an issue at all.

Please create a new Discussion describing the issue that Peacock doesn't launch for you at all, that you are using Windows 11, and any errors you notice in the terminal.

And not by using this discussion, where Peacock _does_ launch, but displays warnings (which is _normal_ for VTK to do) in the terminal.


Edit:

@CPQ-ui, if you are reading this:
I had the privilege to work from home today. My Windows 11 gaming rig is only weeks old. So I had to start fresh and install WSL and Ubuntu 24 using the Microsoft App Store.

Everything worked fine for me:

Installed WSL using powershell:
```bash
wsl --install
```
Then, I chose Ubuntu 24 from the Microsoft App store as my choice of Linux product.
Launched it, ran through the installer instructions that were on the screen, installed Miniforge3 as per our Linux instructions, cloned moose, then ran peacock. It simply launched without error.

Because this is a gaming rig, I already had my NVIDIA drivers installed (install what video drivers you need for your machine).

I do not know what else could be the issue. But because my machine is brand new with so very little installed on it, I can only summarize that something possibly with your WSL environment is not correct.

**Reply by Rforoeverad** [reply]
Thanks.I have deleted my question and use paraview instead.
